web/src/components/software/SoftwarePage.tsx

```diff
@@ -119,7 +119,7 @@ const ReloadSection = ({
       </>
     ) : (
       <>
-        {errorMsg}{" "}
+        {_(errorMsg)}{" "}
```

Comment:
NP: Related to testing, I would love to have a handy collection of repositories broken in various ways: missing/mismatched signatures, keys... we should ask the zypp people or start it ourselves